packages/hash/api/src/auth/seed-dev-users.ts

```diff
@@ -0,0 +1,80 @@
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import { AxiosError } from "axios";
+
+import { GraphApi } from "../graph";
+import { UserModel } from "../model";
+import { adminKratosSdk, KratosUserIdentityTraits } from "./ory-kratos";
+
+/** @todo shortname and fullName is currently unused,
+      these need to be added to the UserModel creation step. */
+type DevelopmentUser = {
+  email: string;
+  shortname: string;
+  fullName: string;
+};
+
+const devUsers: readonly DevelopmentUser[] = [
+  {
+    email: "alice@example.com",
+    shortname: "alice",
+    fullName: "Alice Alison",
+  },
+  {
+    email: "bob@example.com",
+    shortname: "bob",
+    fullName: "Bob Bobson",
+  },
+] as const;
+
+const devPassword = "password";
+
+export const seedDevUsers = async ({
```

Comment:
Can we rename this to something like `ensureDevUsersAreSeeded` to make it more explicit that they aren't seeded when this method is called if they already exist?

Reply:
Addressed in d69bedb18fea3f7ceb5117c0ae722fd4f9ac96e3